cf-custom-resources/lib/copy-assets.js

```diff
@@ -9,10 +9,19 @@ const aws = require("aws-sdk");
  * Main handler, invoked by Lambda
  */
 exports.handler = async function (event, context, callback) {
-	const s3 = new aws.S3();
-	await s3.copyObject({
-		CopySource: event.srcBucket + "/" + event.mapping.path,
-		Bucket: event.destBucket,
-		Key: event.mapping.destPath
-	}).promise();
-};
\ No newline at end of file
+  const s3 = new aws.S3();
+  await s3
+    .copyObject({
+      CopySource: event.srcBucket + "/" + event.mapping.path,
+      Bucket: event.destBucket,
+      Key: event.mapping.destPath,
+      ContentType:
```

Comment:
what happens if this is set to `""`?

Reply:
no error but doesn't make any sense either.